qa/L0_custom_ops/test.sh

```diff
@@ -60,6 +60,7 @@ RET=0
 LD_LIBRARY_PATH=/opt/tritonserver/backends/tensorflow2:$LD_LIBRARY_PATH
 
 # Tensorflow
+## LD_PRELOAD
```

Comment:
is this line needed?

Reply:
I added it to distinguish this sub-test (LD_PRELOAD) vs the one below (Runtime loading via config). Updated the wording to "Load operations via LD_PRELOAD" and "Load operations via model config" to be clearer. Let me know if you'd prefer different wording.